eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -199,7 +261,7 @@ function Add-IssueLabels {
 }
 
 # Will add assignees to existing assignees on the issue
-function Add-IssueAssignees {
+function Add-GithubIssueAssignees {
```

Comment:
Should this be `Set-GitHubIssueAssignees`? Also should it just call `Update-GitHubIssue`

Reply:
I'll update the name. It cannot call `Update-GitHubIssue` though. That is a Patch rather than Post request and the URL is different. Also `Update-GitHubIssue`  replaces the present assignees while this function appends to the current list of assignees.